### PR TITLE
Use item card template in group scroll

### LINKED ISSUE #199: Use item card template in group scroll
There is a standardized template for displaying item cards. It should be used in group item scroll

### PR DESCRIPTION
**Pull request by DieKautz**
This fixes #199

### CODE REVIEW

File: app/views/groups/index.html.erb [hunk 1]
```diff
@@ -18,15 +18,10 @@
...
-        <% [1,2,3,4,5,6,7,8,9].each do |item_num| %>
-        <div class="card col-12 col-md-3 mx-2">
-          <%= image_tag "dummy_image.png", class: "card-img-top" %>
-          <div class="card-body">
-            <h5 class="card-title">Item <%= item_num %></h5>
-              <p class="card-text">von: <span class="text-warning">Herbert Beimel</span></p>
-              <p class="card-text text-success">verfügbar</p>
+        <% group.viewable_items.each do |item| %>
+          <div class="card col-12 col-md-3 mx-2">
+            <%= render "items/item_card", item: item %>
```
Comment (kbroschke) [inline app/views/groups/index.html.erb:23]: Why is `card` class not part of the partial?

Comment (kbroschke) [reply app/views/groups/index.html.erb:23]: Oh, never mind, it is. Then this declaration is redundant. (line 22)

File: app/views/items/_item_card.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            <% end %>
+            <% end %>
+        </ul>
+        <%= render 'items/owner_component', item: item %>
+        <p class="card-text text-truncate text-success"><%= item.status_text(current_user) %></p>
+        <div class="card-body">
+            <a class="mx-auto btn btn-primary" href=<%=item_path(item)%>><%=I18n.t("items.buttons.show")%></a>
+        </div>
+    </div>
+</div>
```
Comment (kbroschke) [inline app/views/items/_item_card.html.erb:18]: Why not use `ViewComponent`?

File: app/views/items/_owner_component.html.erb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div class="row">
     <%# owner / can_manage%>
-    <% owner_groups = Group.owner_groups(@item.id) %>
+    <% owner_groups = Group.owner_groups(item.id) %>
```
Comment (kbroschke) [inline app/views/items/_owner_component.html.erb:3]: Good catch 👌🏻

**Review by kbroschke — CHANGES_REQUESTED** [review]

File: app/views/items/_item_card.html.erb (hunk 2 above)
Comment (DieKautz) [reply app/views/items/_item_card.html.erb:18]: For the "show item" button?
I just moved the card into its own template. Should this be touched in this issue?

File: app/views/groups/index.html.erb (hunk 1 above)
Comment (DieKautz) [reply app/views/groups/index.html.erb:23]: Should be fixed

File: app/views/items/_item_card.html.erb (hunk 2 above)
Comment (kbroschke) [reply app/views/items/_item_card.html.erb:18]: No, it should not.

**Review by kbroschke — APPROVED** [review]

**Pull request merged**